datahub-web-react/src/app/search/SearchablePage.tsx

```diff
@@ -68,6 +70,28 @@ export const SearchablePage = ({ onSearch, onAutoComplete, children }: Props) =>
     const { user } = userContext;
     const viewUrn = userContext.localState?.selectedViewUrn;
 
+    const { title, updateTitle } = useBrowserTitle();
+
+    useEffect(() => {
+        // Update the title only if it's not already set and there is a valid pathname
```

Comment:
@aseembansal-gogo Use case check:

What this code does is it splits the path of the URL itself to determine the title of the page.

For example this will do:

Dataset | Urn:li:sample:etc

For entity pages.

I don't think this is what we want. Please chime in with requirements clarifications.

Reply:
/settings/identities/users -> Settings | Identities | Users